app/views/admin/api_docs/base/swagger.html.erb

```diff
@@ -37,18 +37,70 @@
       }
     }
 
+    function injectAuthParams(spec) {
+      var version = <%= @api_docs_service.service.proxy.authentication_method %>;
+      var authParams;
+      
+      switch(version) {
+        case 2:
+          authParams = [{
+            name: 'x-3scale-auth-app-key',
+            in: 'header',
+            description: 'App Key authentication parameter',
+            'x-data-three-scale-name': 'app_key',
```

Comment:
This should be `app_keys`
https://access.redhat.com/documentation/en-us/red_hat_3scale/2.3/html/api_documentation/create-activedocs-spec